Database/Base/WorldBase.sql

```diff
@@ -102,6 +153,456 @@ CREATE TABLE `points_of_interest` (
+--
+
+DROP TABLE IF EXISTS `recipe_requirements_i_i_d`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `recipe_requirements_i_i_d` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Unique Id of this Recipe Requirement instance',
+  `recipe_Id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'Unique Id of Recipe',
+  `stat` int(10) NOT NULL DEFAULT '0',
+  `value` int(10) unsigned NOT NULL DEFAULT '0',
+  `enum` int(10) NOT NULL DEFAULT '0',
+  `message` text NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `recipe_idx` (`recipe_Id`),
+  CONSTRAINT `recipeId_req_iid` FOREIGN KEY (`recipe_Id`) REFERENCES `recipe` (`recipe_Id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='Recipe IID Requirments';
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `recipe_requirements_int`
+--
+
+DROP TABLE IF EXISTS `recipe_requirements_int`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `recipe_requirements_int` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Unique Id of this Recipe Requirement instance',
+  `recipe_Id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'Unique Id of Recipe',
+  `stat` int(10) NOT NULL DEFAULT '0',
+  `value` int(10) NOT NULL DEFAULT '0',
+  `enum` int(10) NOT NULL DEFAULT '0',
+  `message` text NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `recipe_idx` (`recipe_Id`),
+  CONSTRAINT `recipeId_req_int` FOREIGN KEY (`recipe_Id`) REFERENCES `recipe` (`recipe_Id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='Recipe Int Requirments';
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `recipe_requirements_string`
+--
+
+DROP TABLE IF EXISTS `recipe_requirements_string`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `recipe_requirements_string` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Unique Id of this Recipe Requirement instance',
+  `recipe_Id` int(10) unsigned NOT NULL DEFAULT '0' COMMENT 'Unique Id of Recipe',
+  `stat` int(10) NOT NULL DEFAULT '0',
+  `value` text NOT NULL,
+  `enum` int(10) NOT NULL DEFAULT '0',
+  `message` text NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `recipe_idx` (`recipe_Id`),
+  CONSTRAINT `recipeId_req_string` FOREIGN KEY (`recipe_Id`) REFERENCES `recipe` (`recipe_Id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='Recipe String Requirments';
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `spell`
+--
+
+DROP TABLE IF EXISTS `spell`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `spell` (
```

Comment:
In this table, was the duplicate data that exists in the dat removed from the db?

There's no need to duplicate the data, and, in fact, we must not. The client will be pulling from the dat for those values and the server will need to as well.

Reply:
I made no changes here. before removing the data, I would need know exactly what columns are duplicated and it would also make sense to note what differences, if any, there are to the exported data vs DAT data. I think the coding itself uses the combined data, starting with the DAT then moving over to the DB to fill in the rest but I'm not as well versed in this part as @kdmxp256 or maybe @gmriggs